src/shared_gui_components/WorkflowController.hpp

```diff
@@ -137,6 +137,7 @@ class MeasureStepController : public WorkflowStepController
   int count() override;
 
   MeasureType measureType() const;
+  MeasureLanguage measureLanguage() const;
```

Comment:
Woah, is this a feature now?

Reply:
Yes. Has to be. You need to know if it's a ruby or a python measure! (and the measure manager / workflow run command does need to know too)